lib/tasks/pkg.rake

```diff
@@ -23,6 +23,17 @@ namespace :pkg do
     ref = ENV['ref'] || 'HEAD'
     version = `git show #{ref}:VERSION`.chomp.chomp('-develop')
     raise "can't find VERSION from #{ref}" if version.length == 0
-    `git archive --prefix=foreman-#{version}/ #{ref} | bzip2 -9 > pkg/foreman-#{version}.tar.bz2`
+    `git archive --prefix=foreman-#{version}/ #{ref} > pkg/foreman-#{version}.tar`
+
+    `npm install --no-optional --global-style true`
+    `mv node_modules pkg/`
+
+    Dir.chdir('pkg') do
+      `tar -xvf foreman-#{version}.tar`
+      `mv node_modules/ foreman-#{version}`
+      `tar -cvf foreman-#{version}.tar foreman-#{version}`
```

Comment:
We could I think -- I'll have to change a lot more since this will always produce two tarballs that are required (e.g. changing Jenkins jobs, release jobs, more of the packaging). I can go either way if you have a preference.